docker/Dockerfile.docs

```diff
@@ -13,9 +13,9 @@ RUN for i in 1 2 3; do \
 COPY doc/requirements.txt /tmp/requirements.txt
 RUN pip3 install -r /tmp/requirements.txt
 
-RUN wget https://sourceforge.net/projects/plantuml/files/plantuml.jar/download -O /usr/local/bin/plantuml.jar \
-    && echo '#!/bin/sh\njava -jar /usr/local/bin/plantuml.jar "$@"' > /usr/local/bin/plantuml && \
-    chmod +x /usr/local/bin/plantuml
+RUN wget https://github.com/plantuml/plantuml/releases/latest/download/plantuml.jar -O /usr/local/bin/plantuml.jar \
```

Comment:
This should be a fixed version, not "latest".

Reply:
Latest stable release sounds like a good place to start in this case. We can change it later if necessary.